hack/activate-env.sh

```diff
@@ -36,7 +36,7 @@ _activateEnv() {
   echo "--> Sourcing new environment at $_envFolder..."
   # Need to disable unbound errors for activate
   set +u
-  source $_activate
+  source $_activate &> /dev/null 2>&1
```

Comment:
I would keep the error output visible - in case it fails to activate the environment somehow

Reply:
Updated to be only `1> /dev/null` so that should silence all non-error logging